Tools/CISupport/build-webkit-org/config.json

```diff
@@ -690,11 +692,13 @@
                   {
                     "name": "WPE-Linux-64-bit-Release-WebDriver-Tests", "factory": "TestWebDriverFactory",
                     "platform": "wpe", "configuration": "release", "architectures": ["x86_64"],
+                    "additionalArguments": ["--enable-webdriver-bidi"],
                     "workernames": ["wpe-linux-bot-13"]
                   },
                   {
                     "name": "WPE-Linux-64-bit-Debug-WebDriver-Tests", "factory": "TestWebDriverFactory",
                     "platform": "wpe", "configuration": "debug", "architectures": ["x86_64"],
+                    "additionalArguments": ["--enable-webdriver-bidi"],
```

Comment:
It looks we are enabling WebDriver BiDi on all the bots.

So I wonder if instead of passing the flag here to all the bots, maybe it makes more sense to switch the default in the script `run-webdriver-tests` so it runs with bidi enabled by default (unless `--disable-webdriver-bidi` is passed to it)?

That way we don't need to pass any extra flag here.

Reply:
Good catch. I updated the PR making it opt-out of `run-webdriver-tests`.